### DISCUSSION TITLE
Problems with the way we use  data and GIS

### DISCUSSION THREAD

**Discussion by rachan2023** [albertkun/21S-ASIAAM-191A#19] [Past Thinking Caps]
Data and GIS are useful for presenting information in a clear and concise way to the general public. However, there are institutions and organizations that use data and GIS to support their personal agendas whether that be to encourage others to vote against a political candidate or suppress communities. For example, redlining is used to prevent people who live in an “at risk area” from receiving services, such as mortgage loans to buy a house. The people who created these maps to geographically label which areas are at risk are incorporating their own biases to target certain communities, which contribute to bigger societal problems such as segregation and racism. Another problem with data and GIS is that  these tools can be used to spread false narratives. For example, in the image below, government officials can make claims based on the data below that people are most concerned about the economy, so that they can pass more bills in regards to the economy. People, who may not recognize that all the percentages totaled up exceed 100%, may believe the politician’s narrative. As a result, the issues with data and GIS are that they can be used to hurt communities and spread false information in order for personal gain.  

![Misleading-Graphs-2020-Examples-1](https://user-images.githubusercontent.com/81612098/113367280-72d12100-9310-11eb-8c1f-3392edc12548.png)

**Comment by annyrviloria** [comment]
I really appreciate the point you make about redlining and the very real, deep consequences it has had on people's lives. Maps were quite literally redrawn to keep people of color confined to certain neighborhoods and away from services. It also reminds me of the issue of gerrymandering, where voting districts get redrawn to sabotage the democratic process. I think it is fair to say that these geographic divisions are yet another version of the violence perpetrated by national borders as a result of colonialism and military occupation around the world.
